bitnami/postgresql-ha/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: common
   repository: https://charts.bitnami.com/bitnami
-  version: 1.10.0
-digest: sha256:a8a6f766af11d118c2d8ea1ea7bd670a48ec26721201f47e0c0fac3f586b3290
-generated: "2021-10-07T05:06:05.043787114Z"
+  version: 1.10.1
```

Comment:
Please. keep the Chart.lock as original change. The changes in the Chart.lock are not needed to track. This file is updated automatically by our pipeline.